lib/device_api/android/adb.rb

```diff
@@ -233,6 +233,34 @@ def self.screencap( serial, args )
         shell(serial, cmd)
       end
 
+      def self.check_ip_address(ipaddressandport)
+        unless ipaddressandport =~ /\A(\d{1,3})\.(\d{1,3})\.(\d{1,3})\.(\d{1,3}):[0-9]+\Z/ 
+             raise ADBCommandError.new("Invalid IP address and port " + ipaddressandport)
+        end
+      end
+
+      def self.connect(ipaddressandport)
```

Comment:
Similar to the comment above on the `check_ip_address` method I think this should be:

```ruby
def self.connect(ipaddress, port = 5555)
```